### PR TITLE
Added brief guide for running ReactJS.NET in Azure.

### PR DESCRIPTION
**Pull request by samppis**
ReactJS.NET version 2.2.0 fixes all issues related to running the V8 JavaScript engine in Azure. I still wrote a brief documentation so that people will know it works and that a version update might be necessary. I also included code to check the available JavaScript engines and how to disable the MSIE engine in case someone is still having problems with V8.

Please feel free to modify the text or fix typos etc. if necessary!

### CODE REVIEW

File: site/jekyll/guides/azure.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    return sb.ToString();
+}
+```
+
+To force ReactJS.NET to use the V8 JavaScript engine (and throw an exception if it isn't available) set the AllowMsieEngine configuration property to false.
+
+```csharp
+app.UseReact(config =>
```
Comment (Daniel15) [inline site/jekyll/guides/azure.md]: It's probably worth mentioning that this syntax (`UseReact`) is specific to ASP.NET 5, and users of ASP.NET 4 will need to do this in their `ReactConfig.cs` file intsead.

File: site/jekyll/guides/azure.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+title: Azure
+---
+
+From **version 2.2.0** onwards ReactJS.NET works out of the box in Azure using the V8 JavaScript engine. Versions prior to 2.2.0 will fall back to using MSIE JavaScript engine and you may experience JavaScript errors during server-side rendering that you aren't experiencing locally.
```
Comment (Daniel15) [inline site/jekyll/guides/azure.md]: Nit: Just say `2.2` rather than `2.2.0`, I usually omit the `.0` :)

**Comment by Daniel15** [comment]
This looks pretty good. I just posted some small comments inline. Thank you so much! :smile:

**Comment by Daniel15** [comment]
Thanks, I pushed this to the live site: http://reactjs.net/guides/azure.html

This makes me think that I should probably add a better way to get a list of all the available engines...

**Comment by samppis** [comment]
Great, thanks! That might be a good idea ;)

2016-01-25 14:10 GMT+07:00 Daniel Lo Nigro <notifications@github.com>:

**Pull request merged**